app/frontend/src/components/LoginButton/LoginButton.tsx

```diff
@@ -0,0 +1,40 @@
+import { DefaultButton } from '@fluentui/react';
+import { useMsal } from '@azure/msal-react';
+
+import styles from "./LoginButton.module.css";
+import { loginRequest } from '../../authConfig';
+
+
+export const LoginButton = () => {
+  const { instance } = useMsal();
+  const activeAccount = instance.getActiveAccount();
+  const handleLoginPopup = () => {
+      /**
+       * When using popup and silent APIs, we recommend setting the redirectUri to a blank page or a page
+       * that does not implement MSAL. Keep in mind that all redirect routes must be registered with the application
+       * For more information, please follow this link: https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/login-user.md#redirecturi-considerations
+       */
+      instance
+          .loginPopup({
+              ...loginRequest,
+              redirectUri: '/redirect',
+          })
+          .catch((error) => console.log(error));
+  };
+  const handleLogoutPopup = () => {
```

Comment:
is there a way to wait until the user logs out using the popup? if i X out it "logs out", but when i log in it doesn't prompt me again to log in, it just logs in using the last account automatically.

Reply:
fyi @chuwik , @srbalakr is it OK if i file a followup issue, the login button can be a bit flaky with the popup. it mostly works though

Reply:
Let me add a internal work item to track.